### PR TITLE
[AAE-6309] Cleaning process references

### PR DESCRIPTION
**Pull request by BSekula**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/app/components/main-action/main-action.component.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
         <button
             *ngSwitchCase="actionTypes.button"
             mat-stroked-button
+            [id]="action.id"
+            [disabled]="action.disabled"
```
Comment (mauriziovitale) [inline src/app/components/main-action/main-action.component.html:7]: why do we need these additional params?

**Review by mauriziovitale — CHANGES_REQUESTED** [review]
Can we try to format the files in the same way?

File: src/app/components/main-action/main-action.component.html (hunk 1 above)
Comment (mauriziovitale) [reply src/app/components/main-action/main-action.component.html:7]: can we add unit test to cover this scenario?

Comment (BSekula) [reply src/app/components/main-action/main-action.component.html:7]: Added test for disabled button

**Review by mauriziovitale — APPROVED** [review]

**Pull request merged**